docs/src/manual/gettingStarted.md

```diff
@@ -18,6 +18,8 @@ push!(win,b)
 ```
 Finally, `showall(win)` will render the entire application on the screen.
 
+Note that if you run the above example and only a blank window renders, see the `Common Issues` section [here](https://juliagraphics.github.io/Gtk.jl/latest/index.html). 
```

Comment:
You can use internal link in docs: `(../index.md)`
Since #435 add `## Common Issues` you can add it to your link: `(../index.md#common-issues)` should work. Build the doc and try locally to be sure

Reply:
Done!